### ISSUE TITLE
SRT file not displaying subtitles on OSX/iOS

### ISSUE THREAD

**Issue by IARI** [RenderHeads/UnityPlugin-AVProVideo#678]
**Describe the bug**
We want to use subtitles (srt sideloading) in our mobile Application.
Sideloading .srt Files does not work on OSX and iOS.

Everything does work as expected 
* in the Windows Editor
* on Android builds (Samsung S8)

However, no SubtitleChange events appear to be fired
* in the OSX Editor 
* on iOS builds (iPhone XR)

To check if it does work, i use both
* a `SubtitlesUGUI` component
* an Eventhandler with Debug.Log like this:
```C#
    private void OnMediaPlayerEvent(MediaPlayer arg0, MediaPlayerEvent.EventType e, ErrorCode arg2) {
        switch (e) {
            // ...
            case MediaPlayerEvent.EventType.SubtitleChange:
                Debug.Log($"Subs: {MediaPlayer.Subtitles.GetSubtitleText()}");
                break;
            // ...
        }
    }
```

**Additional info:**
Our Videos and srt files are included at the location `StreamingAssets/videos` in the Project.
On iOS and Android behaviour is slightly different:
* Android: we first copy videos and to a persistentDataPath. Then we load Videos and subtitles form there.
* iOS: we directly load videos and subtitles from `StreamingAssets/videos`
However, adjusting the behaviour on iOS to be exactly the same did not help - subtitles are still broken on OSX/iOS.

**Your Setup (please complete the following information):**
 - Unity version: 2019.4.10f1
 - AVPro Video version: 2.0.6
 - Operating system version:  see bug description.
 - Device model: see bug description.
 - Video specs (resolution, frame-rate, codec, file size):  (shouldn't be relevant)

**To Reproduce**
On OSX Editor (and iOS build)
1. Place Video and SRT files below persistentDataPath (or StreamingAssets - doesnt matte for me)
2. make sure that variables `videoPath` and `srtPath` hold the correct paths to the files.
2. Load Media
```C#
MediaPlayer.SideloadSubtitles = true;
var loadedSubtitles =
    MediaPlayer.EnableSubtitles(new MediaPath( srtPath  , MediaPathType.AbsolutePathOrURL));
if (!loadedSubtitles) {
    Debug.LogWarning($"Failed to load subtitles."); // this does not happen on any platform.
}
MediaPlayer.OpenMedia(MediaPathType.AbsolutePathOrURL, videoPath);
```

**Logs**
No errors were logged.

**Srt sample file**
[test.zip](https://github.com/RenderHeads/UnityPlugin-AVProVideo/files/6140688/test.zip)

**Comment by MorrisRH** [comment]
This is fixed in the latest release 2.0.10

**Comment by IARI** [comment]
Thanks,

I have tested both on an OSX and an iOS Device - and in both instances subtitles now appear to be working :)
closing the issue.
